graphql-dgs-spring-webmvc-autoconfigure/src/main/resources/META-INF/additional-spring-configuration-metadata.json

```diff
@@ -0,0 +1,16 @@
+{
+  "properties": [
```

Comment:
Shouldn't `dgs.graphql.path` be included as well?

Reply:
I split out the path config based on our earlier discussion.